api/routers/legacy_debug.py

```diff
@@ -124,7 +125,21 @@ async def detect(detections:List[detection], manual_detect:int) -> None:
     # data validation, there can only be one reporter, and it is unrealistic to send more then 5k reports.
     if len(df) > 5000 or df["reporter"].nunique() > 1:
         logger.debug('Too many reports.')
-        return {'NOK': 'NOK'}, 400
+        return {'ERROR': 'ERROR'}, 400
```

Comment:
I don't actually think we use this line anymore besides the return itself. I like having ERROR next to it in all-caps though. XD

Reply:
lmaoo yeah